rgw/admin/bucket.go

```diff
@@ -77,7 +77,11 @@ type Policy struct {
 		} `json:"grant_map"`
 	} `json:"acl"`
 	Owner struct {
-		ID          string `json:"id"`
+		ID struct {
+			ID     string `json:"id"`
+			Tenant string `json:"tenant"`
+			NS     string `json:"ns"`
+		} `json:"id"`
```

Comment:
I see that the reporter complaining about mismatch in owner type for both Bucket and Policy struct. Can we check whether owner type is correct for Bucket struct too?

Reply:
Atleast from the docs owner is a string type https://docs.ceph.com/en/latest/radosgw/adminops/#get-bucket-info, not a container